utils/node_local_test.py

```diff
@@ -6327,6 +6327,23 @@ def server_fi(args):
         server.set_fi(probability=0)
 
 
+def look_for_library_call(conf, cmd, library_str):
+    """Look for library_str in the strace call stack of running cmd."""
+    tmpfile = tempfile.NamedTemporaryFile(mode='r',
```

Comment:
I don't recall if pylint will still complain on every PR thereafter but it used to be the case.  You could either remove strace from the comment or add a comment such as https://github.com/daos-stack/daos/blob/b5d2047a7283b25d378562fcb8152d3e72b6d6e5/utils/node_local_test.py#L3402